examples/regexp-type-langlib-functions/regexp_langlib_functions.bal

```diff
@@ -0,0 +1,39 @@
+import ballerina/lang.regexp;
+import ballerina/io;
+
+public function main() returns error? {
+
+    string data = "bob@example.net,alice@example.com,charlie@example.org,david@example.xyz,emma@example.io";
+
+    // Split the comma separated email list
+    string[] emails = re `,`.split(data);
+
+    // Print the list of emails
+    io:println(emails);
+
+    // Valid email pattern
+    string:RegExp validEmailPattern = re `[a-z]+@[a-z]+\.[a-z]+`;
```

Comment:
Shall we use the original regex, instead of this one.
ex:- `([a-zA-Z0-9._%+-])+@([a-zA-Z0-9.-])+\.([a-zA-Z]{2,})`